mesonbuild/compilers/detect.py

```diff
@@ -616,7 +616,7 @@ def detect_cpp_compiler(env: 'Environment', for_machine: MachineChoice) -> Compi
     return _detect_c_or_cpp_compiler(env, 'cpp', for_machine)
 
 def detect_cuda_compiler(env: 'Environment', for_machine: MachineChoice) -> Compiler:
-    from .cuda import CudaCompiler
+    from .cuda import CudaCompiler, _Phase
```

Comment:
If you're going to take `_Phase` outside of cuda, it needs to be renamed to `Phase`. Python uses a leading `_` to mean "protected".

Reply:
Changed accordingly.